resources/functionMap.php

```diff
@@ -11692,14 +11692,14 @@
 'stomp_version' => ['string'],
 'StompException::getDetails' => ['string'],
 'StompFrame::__construct' => ['void', 'command='=>'string', 'headers='=>'array', 'body='=>'string'],
-'str_getcsv' => ['array', 'input'=>'string', 'delimiter='=>'string', 'enclosure='=>'string', 'escape='=>'string'],
+'str_getcsv' => ['non-empty-list<?string>', 'input'=>'string', 'delimiter='=>'string', 'enclosure='=>'string', 'escape='=>'string'],
```

Comment:
Why is the value nullable? I've got a related bug report around this: https://github.com/phpstan/phpstan/issues/9393

Reply:
https://3v4l.org/EJhab

Reply:
Thank you!